back-end/hub-api/src/test/java/io/apicurio/hub/api/gitlab/GitLabSourceConnectorTest.java

```diff
@@ -217,7 +217,7 @@ public void testGetProjects() throws GitLabException, SourceConnectorException {
     @Test
     @Ignore
     public void testGetBranches() throws GitLabException, SourceConnectorException {
-        Collection<SourceCodeBranch> branches = service.getBranches("Apicurio", "api-samples");
+        Collection<SourceCodeBranch> branches = service.getBranches( "api-samples");
```

Comment:
I'm assuming I will need to lookup the unique projectId for the `Apicurio/api-samples` project on GitLab, yes?

Reply:
Otherwise this will not work?

Reply:
Yes. That would be correct.